GeoDaCenter/chicago-environment-explorer
# feat: added Displacement Pressure variable + links to data

## Problem
New data has been provided for Displacement Pressure - we would like to integrate this into the existing ChiVes map data

Fixes #117 

## Approach
* Added new data sheet link to Sheets
* Added `VUL_PRICE`, `VULNERABLE`, and `CHANGE` to Columns
* Added `Displacement Pressure` to Variables
* Adjust `MapSection` + `Legend` to support new variables / values

### TODOs
* Verify map / data correctness
* Description for new "Displacement Pressure" variable
* Replace `0` with `Not Vulnerable`?
* Use the same color scale as the previous data? Exclude 0 from color scale? Can this color be transparent?

## How to Test
1. Navigate to https://deploy-preview-127--chicago-env-explorer.netlify.app/map
2. On the left side, expand the list of Variables
3. Scroll down and select "Displacement Pressure" from the list - this is the new variable + data
    * You should see the new variable data / scale is displayed on the map
